pkg/minikube/tunnel/route_darwin.go

```diff
@@ -162,5 +167,37 @@ func (router *osRouter) Cleanup(route *Route) error {
 	if !re.MatchString(message) {
 		return fmt.Errorf("error deleting route: %s, %d", message, len(strings.Split(message, "\n")))
 	}
+	// idempotent removal of cluster domain dns
+	resolverFile := fmt.Sprintf("/etc/resolver/%s", route.ClusterDomain)
```

Comment:
Make '/etc/resolver' a constant to share between functions, and use filepath.Join to create paths from it.

Also, I prefer that the variable name uses path instead of file, since the latter references something specific in Go (file.File)